RealEstate/apps/core/views.py

```diff
@@ -296,6 +332,16 @@ def post(self, request, *args, **kwargs):
                             content_type="application/json")
 
 
+class PasswordChangeDoneView(BaseView):
+    """
+    This is needed to provide a message to the user that their password change
+    was successful.
+    """
+    def get(self, request, *args, **kwargs):
+        messages.success(request, "Password change successful")
```

Comment:
Chris had a success message which was showing earlier that "you  have successfully change your password" but it was showing on homebuyer dashboard page. I think it should show underneath or top of the password change page.